.github/workflows/test-and-demo.yaml

```diff
@@ -108,6 +112,7 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     needs: setup
+    if: ${{ github.repository_owner == 'wikit' }}
```

Comment:
I think this broke our CI. It should have run in #443, but it didn't.

Reply:
Wait, shouldn’t this be `wmde`? `wikit` is the repository _name_, not _owner.

Reply:
Let’s see if #444 fixes it.